packages/mongo/mongo_driver.js

```diff
@@ -7,6 +7,8 @@
  * these outside of a fiber they will explode!
  */
 
+var path = require("path");
```

Comment:
`var` here, `const` down bellow. What version of ES are we using in this file?

Reply:
I believe it's a mix :( I'm going to update this anyway.